### ISSUE TITLE
Windows 10 low memory issues

### ISSUE THREAD

**Issue by Roaders** [nanopool/Claymore-Dual-Miner#122]
Hi All

I have my miner running on a Windows 10 machine that has 8GB of memory on it

![image](https://user-images.githubusercontent.com/10414642/31861938-8b3b8284-b72d-11e7-85d1-a50de9648e6b.png)

This is a new windows install with nothing else installed other than AVG and is only used for mining.

When I come back to the PC I find an alert saying that the PC has low memory and advising that I should close down `EthDcrMiner64.exe`.

I thought that 8 GB was plenty for mining.

Do I need to worry about this? Can I do anything to prevent the alert?

I currently have 7 8GB RX 480 cards, 1 4GB RX 480 card and 1 8 GB 1070 card.

Thanks

**Comment by DeviaVir** [comment]
See "Increase Virtual Memory" on this page:
http://1stminingrig.com/best-windows-setup-configuration-tweaks-for-mining/

You may also want to pull out the 1070, as it means you need to run two miners (OpenCL vs CUDA) which drains your system resources further.

**Comment by Roaders** [comment]
Yup, I already have 16GB of virtual memory setup... Looks like there are some other helpful tweaks on that page though. Thanks
